Dockerfile

```diff
@@ -67,5 +67,5 @@ ADD src /app/src
 ADD fetch_raw_data.py /app
 #ADD fetch_flow_definitions.py /app
```

Comment:
do we need this too? Sorry I didn't review the earlier files so I am not sure.

Reply:
We don't need to uncomment it now, because this file isn't currently present in the project or necessary for WorldBank.

It's inherited from an older experiment that never got deployed. Let me ask again on the DT channel if we still need this, then we can decide whether to reactivate or remove (in a future PR)